src/common/Map/AppMap.tsx

```diff
@@ -44,6 +50,57 @@ const Map: React.FC<MapProps> = ({ height }) => {
     style,
   } = settings;
 
+  const baseURL = 'http://localhost:5000/api/cafe/list';
+
+  // const getCafeList = async () => {
+  //   let response
+  //   try {
+  //     response = await axios.get(baseURL);
+  //     const fetchData = await response.data;
+  //     setListCafes(fetchData);
+  //     setIsLoading(true);
+  //   }
+  //   catch (error) {
+  //     console.log(error.message);
+  //   }
+
+  //   if (response === null) {
+  //     return;
+  //   }
+  // }
+
+  useEffect(() => {
+    const getCafeList = async () => {
+      let response
+      try {
+        response = await axios.get(baseURL);
+        const fetchData = await response.data;
+        setListCafes(fetchData);
+        setIsLoading(true);
```

Comment:
tady by asi melo byt setIsLoading(false)
a nastavujes tady dva states najednou, bylo by lepsi udelat nejaky state object a upravit to jednim setStatem